BrandonBeltran/hackerrank/03/JavaMap.java

```diff
@@ -0,0 +1,34 @@
+package bootcamp;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class JavaMap {
+
+    public static void main(String[] args) {
+
+        Map<String, Integer> phoneBook = new HashMap<>();
+
+        Scanner scanner = new Scanner(System.in);
+        int queries = Integer.parseInt(scanner.nextLine());
+        for(int iterator = 0; iterator < queries; iterator++){
+
+            String name = scanner.nextLine();
+            int number = Integer.parseInt(scanner.nextLine());
+            phoneBook.put(name, number);
+
+        }
+
+        while(scanner.hasNext()){
+            String query = scanner.nextLine();
+            if(phoneBook.containsKey(query)){
```

Comment:
perfect!! don't forget Collections has done -almost- all work for us, sometimes you only need to find it in API doc